stdlib/public/Darwin/Foundation/Data.swift

```diff
@@ -62,14 +62,14 @@ internal func __NSDataIsCompact(_ data: NSData) -> Bool {
 
 #endif
 
+// Underlying storage representation for medium and large data.
+// Inlinability strategy: methods from here should not inline into InlineSlice or LargeSlice unless trivial.
```

Comment:
Every type I audited now has a description of what inlinability we're aiming for.